### ISSUE TITLE
Generated Images Have Some Blocking Artifact

### ISSUE THREAD

**Issue by wsxtyrdd** [VITA-Group/TransGAN#36]
Due to the patch-wise generation of TransGAN, I found some blocking artifacts in your generations. I think the authors had already known these phenomena. Are there any tricks to eliminate these artifacts?
![image](https://user-images.githubusercontent.com/43609399/124730513-020b5780-df44-11eb-972d-e24fb9754d2e.png)

**Comment by yueruchen** [comment]
Yes, we have noticed that and found that strong data augmentation and long training will help. If you check the training script you might find the current training code includes multiple data augmentations. The above image is still generated under simple data augmentation, I will replace it soon.

**Comment by RSMung** [comment]
Does the trick: long training means that even though the loss of model has been stable, we will continue training our model for eliminate these artifacts?
My second understanding of long training is that just because of data augmentations, we have to cost more time to train our model.
Which understanding is right?
Waiting for your reply, thanks.

**Comment by yifanjiang97** [comment]
@RSMung My feeling is GAN loss can not indicate whether the training converges or not, because it is adversarial learning. That means, even the training loss is stable your generated images may still not in the perfect condition. So usually I do not use training loss to judge whether I should finish the training, instead I directly see the FID scores. 

And yes, strong data augmentation requires more training iterations.

**Comment by RSMung** [comment]
Thank you for your detailed explanation!
